### PR TITLE
Align the service broker install instructions with upstream

### PR DESCRIPTION
**Pull request by satadruroy**
Updated the README to align with upstream AWS SB install instructions.

### CODE REVIEW

File: aws-servicebroker/README.md [hunk 1]
```diff
@@ -80,72 +22,59 @@ name (or, alternatively, provide an `AWS_VPC` override to `make`).
...
+AWS service broker make sure to update the helm chart version (the version as of this
+writing is 1.0.0-beta.3). For the broker install, pass in a value indicating the Cluster
+Service Broker should not be installed.
+
+`helm install aws-sb/aws-servicebroker --name aws-servicebroker --namespace aws-sb --version 1.0.0-beta.3 --set aws.secretkey=$aws_access_key --set aws.accesskeyid=$aws_key_id --set deployClusterServiceBroker=false --set tls.cert="$(base64 -w0 tls.pem)" --set tls.key="$(base64 -w0 tls.key)" --set aws.tablename=awssb --set aws.vpcid=$vpcid --set aws.region=$aws_region --set authenticate=false`
+
+6.Create a service broker in CF:
   ```
-	cf create-service-broker aws unused-username unused-password  https://aws-servicebroker.${BROKER_NAMESPACE}
+	cf create-service-broker aws-servicebroker unused-username unused-password  https://aws-servicebroker.${BROKER_NAMESPACE}
```
Comment (btat) [inline aws-servicebroker/README.md:73]: In the `helm install` command `${BROKER_NAMESPACE}` has been set to `aws-sb` whereas other commands (like this one) still use the variable.

File: aws-servicebroker/README.md [hunk 2]
```diff
@@ -80,72 +22,59 @@ name (or, alternatively, provide an `AWS_VPC` override to `make`).
...
 		--region ${AWS_REGION} --table-name awssb
 ```
-2. Wait for the table to finish creating.
-3. Create a server certificate for the broker (substitute `CF_NAMESPACE` and `BROKER_NAMESPACE`):
-    1. Get the CA certificate:
+3.Wait for the table to finish creating.
+
+4.Create a server certificate for the broker (substitute `CF_NAMESPACE` and `BROKER_NAMESPACE`):
+
+    a. Get the CA certificate:
```
Comment (cjschroder) [inline aws-servicebroker/README.md]: Not sure what happened here with the markup -- when I look at build view (https://github.com/SUSE/cf-brokers/tree/sroy/update-docs-use-upstream-aws-sb/aws-servicebroker), the whole block from a to e is grayed out as a code block, and the backticks (e.g. the kubectl commands) inside the block are visible.

Comment (cjschroder) [reply aws-servicebroker/README.md]: Overall this looks good, nice and clean and understandable.

**Comment by satadruroy** [comment]
Thanks for the review comments which I have addressed. If it looks ok, can you squash and merge please?

**Comment by btat** [comment]
LGTM

**Pull request merged**